test/models/run_test.exs

```diff
@@ -15,4 +17,12 @@ defmodule Gatherto.RunTest do
     changeset = Run.changeset(%Run{}, @invalid_attrs)
     refute changeset.valid?
   end
+
+  describe "persistance" do
+    test "a run with valid attributes" do
+      Run.changeset(%Run{}, @valid_attrs) |> Repo.insert!
+
+      assert Repo.get_by!(Run, @valid_attrs)
+    end
+  end
```

Comment:
I like it. We can let the test fluctuate organically as we learn the framework and get a better understanding of what needs to be tested and what is covered elsewhere.